doc/tutorials/pre-launched-rt.rst

```diff
@@ -30,60 +30,60 @@ Example of Pre-Launched RT
 Take the Whiskey Lake WHL-IPC-I5 board (as described in :ref:`hardware`) for
 example. This platform can connect both an NVMe and a SATA drive and has
-two Ethernet ports. We will passthrough the SATA and Ethernet 03:00.0
-devices into the Pre-Launched RT VM, and give the rest of the devices to
+two Ethernet ports. We will pass through the SATA and Ethernet 03:00.0
+devices into the pre-launched RTVM, and give the rest of the devices to
 the Service VM.
 
-Install SOS With Grub on NVMe
-=============================
+Install Service VM OS With GRUB on NVMe
+=======================================
 
-As with the Hybrid and Logical Partition scenarios, the Pre-Launched RT
+As with the Hybrid and Partition scenarios, the Pre-Launched RT
 mode must boot using GRUB.  The ACRN hypervisor is loaded as a GRUB
-multiboot kernel, while the Pre-Launched RT kernel and Service VM
-kernels are loaded as multiboot modules. The ACRN hypervisor, Service
-VM, and Pre-Launched RT kernel images are all located on the NVMe drive.
+multiboot kernel, while the pre-launched RTVM kernel and Service VM
+kernel are loaded as multiboot modules. The ACRN hypervisor, Service
+VM, and pre-launched RTVM kernel images are all located on the NVMe drive.
 We recommend installing Ubuntu on the NVMe drive as the Service VM OS,
 which also has the required GRUB image to launch Pre-Launched RT mode.
-Refer to :ref:`gsg`, to
-install Ubuntu on the NVMe drive, and use grub to launch the Service VM.
+Refer to :ref:`gsg` to
+install Ubuntu on the NVMe drive, and use GRUB to launch the Service VM.
 
 Install Pre-Launched RT Filesystem on SATA and Kernel Image on NVMe
 ===================================================================
 
-Follow the :ref:`gsg` to install RT rootfs on SATA drive.
+Follow the :ref:`gsg` to install RT rootfs on the SATA drive.
 
-The Kernel should
+The kernel should
 be on the NVMe drive along with GRUB. You'll need to copy the RT kernel
 to the NVMe drive. Once you have successfully installed and booted
 Ubuntu from the NVMe drive, you'll then need to copy the RT kernel from
 the SATA to the NVMe drive:
 
 .. code-block:: none
 
-   # mount /dev/nvme0n1p1 /boot
-   # mount /dev/sda1 /mnt
-   # cp /mnt/bzImage /boot/EFI/BOOT/bzImage_RT
+   sudo mount /dev/nvme0n1p1 /boot
+   sudo mount /dev/sda1 /mnt
+   sudo cp /mnt/bzImage /boot/EFI/BOOT/bzImage_RT
 
 Build ACRN With Pre-Launched RT Mode
 ====================================
 
 The ACRN VM configuration framework can easily configure resources for
-Pre-Launched VMs. On Whiskey Lake WHL-IPC-I5, to passthrough SATA and
-Ethernet 03:00.0 devices to the Pre-Launched RT VM, build ACRN with:
+pre-launched VMs. On Whiskey Lake WHL-IPC-I5, to pass through SATA and
+Ethernet 03:00.0 devices to the pre-launched RTVM, build ACRN with:
 
 .. code-block:: none
 
    make BOARD_FILE=$PWD/misc/acrn-config/xmls/board-xmls/whl-ipc-i5.xml SCENARIO_FILE=$PWD/misc/acrn-config/xmls/config-xmls/whl-ipc-i5/hybrid_rt.xml RELEASE=0
```

Comment:
The file paths look outdated, assuming they are referring to the acrn-hypervisor repo.

Reply:
Good catch, we should replace this line with:
```
make BOARD=whl-ipc-i5 SCENARIO=hybrid_rt
```